xml/System.Runtime.InteropServices/Marshal.xml

```diff
@@ -3501,7 +3501,7 @@
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- The delegate `d` is converted to a function pointer that can be passed to unmanaged code using the [__stdcall](https://msdn.microsoft.com/library/e212594b-1827-4d07-9527-7d412b300df8) calling convention.  
+ The delegate `d` is converted to a function pointer that can be called by unmanaged code using the calling convention specified by the <xref:System.Runtime.InteropServices.UnmanagedFunctionPointerAttribute> associated with the delegate's type or [__stdcall](/cpp/cpp/stdcall) if none is specified.
```

Comment:
Please indicate the default first and then the way to modify the default. Note that the usage of `__stdcall` is only for x86, the [x64 calling convention](https://docs.microsoft.com/cpp/build/x64-calling-convention) is used on x64 platform.